### DISCUSSION TITLE
Regression: `Quote reply` doesn't work for review comments

### DISCUSSION THREAD

**Discussion by michaelfig** [community/community#8350] [Pull Requests]
When I click the three-dots menu on a PR review comment and choose "Quote reply", it only inserts:

```

>
```

in the new comment box.

I expect it to inserting the highlighted text from the comment, like:

```
> I don't see PASS_STYLE in index.js
```

This is a recent regression, as it worked correctly a few weeks ago.

Here is an example, taken from https://github.com/Agoric/agoric-sdk/pull/4144#pullrequestreview-821537947

https://user-images.githubusercontent.com/457244/144456785-feabd229-b4df-43e2-ab59-27ac0ffdb16e.mov

**Comment by darabos** [comment]
The same thing started happening to me maybe a week or two ago. Sometimes it works, sometimes I get only a `>`. Right now I have two PRs open in two tabs. In one I get the issue in the other I don't. There's no error in the JavaScript console. I'm on Chrome 96 on macOS Big Sur.

**Comment by richvdh** [comment]
This doesn't seem to be limited to PR review comments to me - it happens on comments everywhere.

It's unclear what causes it. It might be to do with whether the comment includes formatting?

**Comment by yeliu84** [comment]
I'm still facing this issue everyday, any updates?

**Comment by BlaineEXE** [comment]
I'm facing this issue also and have been for quite some time. The first time I use "Quote Reply" it only inserts `>`. The second time I use "Quote Reply" it inserts the text I expect.

**Reply by totallyquiche** [reply]
Same for me

**Reply by Esqarrouth** [reply]
same for me. This has been annoying me for many months now

**Reply by rabusmccaleb** [reply]
testing

**Comment by Spacexowboy** [comment]
Stop

**Comment by rsmith20** [comment]
Still bugged exactly as described.

**Comment by BlaineEXE** [comment]
623 days and counting waiting for anyone at GitHub to respond

**Reply by BlaineEXE** [reply]
@michaelfig are you able to add the `bug` label to this? Maybe that would encourage more focus from GH

**Reply by BlaineEXE** [reply]
Also, just tried this again with my comment [here](https://github.com/orgs/community/discussions/8350#discussioncomment-2454915) and got the ridiculous quoted text below:

```
> been
```

I had just freshly reloaded the page and had no text highlighted, and GitHub just picked a random single word in the box and added it. What a joke.

**Comment by BlaineEXE** [comment]
I hit this issue again today. I hit this issue multiple times per day when reviewing Pull requests for github.com/rook/rook, and it has driving me up the wall for over a year now. @willsmythe @queenofcorgis @aspeckofdustin I see that you all may be GitHub team members. Could you please help me (and all of us) get this placed into a bug queue to be fixed?

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time. Thank you ❤️

**Reply by BlaineEXE** [reply]
Apologies. I see in the CoC that there is a [Support contact form](https://support.github.com/contact?tags=docs-community-support) which I will use to attempt to escalate the issue at hand since the issue has gone without response for over a year.
